src/schema/schema_open.c

```diff
@@ -551,12 +551,13 @@ __wt_schema_get_colgroup(WT_SESSION_IMPL *session,
 			if (tablep != NULL)
```

Comment:
The `if` check can be removed, the `else` block can be removed. Per the assertion at the top of this function, `tablep` cannot be NULL.